{
	"check": "at least one",
	"comment": "python38-urllib3 is not installed",
	"id": "oval:com.redhat.unaffected:tst:202015801054",
	"version": "635",
	"check_existence": "none_exist",
	"object": {
		"object_ref": "oval:com.redhat.cve:obj:202128861042"
	},
	"state": {}
}
